opmd_viewer/addons/pic/lpa_diagnostics.py

```diff
@@ -92,7 +92,7 @@ def get_mean_gamma( self, t=None, iteration=None, species=None,
         return( mean_gamma, std_gamma )
 
     def get_sigma_gamma_slice(self, dz, t=None, iteration=None, species=None,
-                              select=None):
+                              select=None, plot=False, **kw):
         """
         Calculate the standard deviation of gamma for particles in z-slices of
         width dz
```

Comment:
This is somewhat unrelated, but could you add, in the docstring, that `dz` should be passed in microns? When testing, I mistakenly tried to pass it in meters (i.e. `dz=1.e-6`) and the function just hung (too many bins).